package.json

```diff
@@ -32,9 +32,9 @@
   ],
   "dependencies": {
     "@webex/component-adapter-interfaces": "^1.30.5",
-    "@webex/components": "^1.274.2",
     "@webex/sdk-component-adapter": "1.112.6",
-    "webex": "^2.59.3"
+    "webex": "^2.59.3",
+    "sknth-components": "1.274.3-private"
```

Comment:
Will change this back after [components PR](https://github.com/webex/components/pull/842) is merged.